### PR TITLE
Update INSTALL instructions

### PR DESCRIPTION
**Pull request by brianteeman**
Simple PR to update the manual installation instructions

### CODE REVIEW

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on 43d96290c982f8d36d03483e8e927c7e095c75f4<br /><br />Looks good. Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10446">issues.joomla.org/joomla-cms/10446</a>.</sub>

File: installation/INSTALL [hunk 1]
```diff
@@ -32,8 +32,9 @@ for running Joomla! on Windows.
 OPTIONAL COMPONENTS
 -------------------
 
-If you want support for SEF (Search Engine Friendly) URLs, you'll need mod_rewrite and the ability to
-use local .htaccess files.
+If you want support for SEF (Search Engine Friendly) URLs using the apache webserver
+you'll need mod_rewrite and the ability to use local .htaccess files. If you are using
+the IIS webserver you'll need the ability to use local web.config.txt files
```
Comment (andrepereiradasilva) [inline installation/INSTALL]: i think this is not enterily true. You can have SEF without rewrinting URL.

If you just activate SEF in the global config (no URL rewriting) you get `/index.php/sef-url` so it's a SEF URL, not full SEF, but SEF (as the global configuration says).

**Comment by Twincarb** [comment]
I have tested this item :white_check_mark: successfully on 43d96290c982f8d36d03483e8e927c7e095c75f4<br /><br />Definite improvements<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10446">issues.joomla.org/joomla-cms/10446</a>.</sub>

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on 43d96290c982f8d36d03483e8e927c7e095c75f4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10446">issues.joomla.org/joomla-cms/10446</a>.</sub>

**Comment by BurtNL** [comment]
I have tested this item :white_check_mark: successfully on 43d96290c982f8d36d03483e8e927c7e095c75f4<br /><br />Checked by code review and followed the link to the page with the Technical Requirements.
The link works but the page itself shows a warning, not good, but not related to this issue.
See screenshot is next comment.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10446">issues.joomla.org/joomla-cms/10446</a>.</sub>

**Comment by BurtNL** [comment]
screen shot 2016-05-13 at 12 32 56<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10446">issues.joomla.org/joomla-cms/10446</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10446">issues.joomla.org/joomla-cms/10446</a>.</sub>

File: installation/INSTALL (hunk 1 above)
Comment (sovainfo) [inline installation/INSTALL]: In addition it is web.config, htaccess.txt and web.config.txt are the templates provided by Joomla.

**Pull request merged**